src/python/dxpy/scripts/dx.py

```diff
@@ -4778,7 +4798,7 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
     parents=[env_args],
     prog='dx list database files'
 )
-parser_list_database_files.add_argument('database', help='ID of the database.')
+parser_list_database_files.add_argument('database', help='Data object ID or path of the database.')
```

Comment:
Maybe change to `Data object ID or name to access` to be consistent with `get` help?

Reply:
I feel as if calling it name might be misleading since it's a fullpath as opposed to just the file name.

Reply:
I'll leave the decision about updating the help on @kpjensen or @emiloslavsky .

Reply:
Path is used in various help output, I think it's fine in this context.